### PR TITLE
fix(component): Updated Network Selector Component Style

### PR DESCRIPTION
**Pull request by thefiend**

### CODE REVIEW

File: src/components/NetworkSelectorContainer.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 import { connect } from "react-redux";
 import { updateWeb3, getNetwork, getCustomRpc } from "../reducers/application";
 import { NETWORK_TYPES } from "../config";
 
+const divBase = css`
+  border-radius: 50px;
+  padding: 0.2em 0.5em;
+`;
+
+const selectorBase = css`
```
Comment (Nebulis) [inline src/components/NetworkSelectorContainer.js]: selectorBase ? or selectBase ?

It looks like css reset, wanna apply it lobally instead ?

Comment (thefiend) [reply src/components/NetworkSelectorContainer.js]: this is just the base style for the selector

Comment (thefiend) [reply src/components/NetworkSelectorContainer.js]: Added to Global CSS in Meta.js, please review again.

File: src/components/NetworkSelectorContainer.js [hunk 2]
```diff
@@ -21,22 +28,19 @@ class AdminContainer extends Component {
...
+            className="pa2 provider-selector"
+            value={this.props.network}
+            onChange={this.handleNetworkChange}
+          >
+            <option value={INJECTED}>Metamask</option>
+            <option value={LEDGER_MAIN}>Ledger Nano (Mainnet)</option>
+            <option value={LEDGER_ROPSTEN}>Ledger Nano (Ropsten)</option>
+          </select>
+        </div>
+      </React.Fragment>
```
Comment (sumitnitsurat) [inline src/components/NetworkSelectorContainer.js]: dont need <React.Fragment> for this case.

Comment (thefiend) [reply src/components/NetworkSelectorContainer.js]: Updated.

**Pull request merged**